css/desktop.css

```diff
@@ -0,0 +1,179 @@
+@media screen and (min-width: 768px) {
+  .headline,
+  .about-me {
+    background-image: none;
+  }
+
+  .about-me__heading::before,
```

Comment:
Keep doing the great job 👍

Reply:
Oh thank you. I have corrected the README file. I'm just going to ask for the next review. Cheers.